pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -395,8 +401,23 @@ func (adc *attachDetachController) GetDesiredStateOfWorld() cache.DesiredStateOf
 }
 
 func (adc *attachDetachController) podUpdate(oldObj, newObj interface{}) {
```

Comment:
I would assume that, `podUpdate` should get called when pod's status for example, changes from `Running` to `Complete` - not every 12 hours.